contrib/cirrus/setup_environment.sh

```diff
@@ -142,6 +142,25 @@ case "$CI_DESIRED_NETWORK" in
     *)          die_unknown CI_DESIRED_NETWORK ;;
 esac
 
+# Database: force SQLite or BoltDB as requested in .cirrus.yml.
+# If unset, will default to BoltDB.
+case "$CI_DESIRED_DATABASE" in
```

Comment:
shellcheck is dumb, it doesn't know that this var is defined in `.cirrus.yml`.  You can bypass the check by placing a `# shellcheck disable=SC2154` line right above the `case` line.

Reply:
Done